assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/stable_diffusion/constants.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+
+class _CustomEnum(Enum):
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
+
+
+class Tasks(_CustomEnum):
+    """
+    Task types supported by stable diffusion
+    """
+    TEXT_TO_IMAGE = "text-to-image"
+    INPAINTING = "inpainting"
+
+
+class COLUMN_NAMES:
+    """
+    Column names in pandas dataframe used to receive request and send response.
+    """
+    TEXT_PROMPTS = "text_prompts"
+    GENERATED_IMAGES = "generated_images"
+
+class DATATYPE_LITERALS:
+    """
+    Literals related to data type.
+    """
+    IMAGE_FORMAT = "JPEG"
```

Comment:
Is it possible to have multiple image formats?

Reply:
i have tested with png as well. It worked